gradle.properties

```diff
@@ -1,10 +1,10 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Quilt properties
-minecraft_version=1.17.1
-yarn_mappings=29
-loader_version=0.11.6
-fabric_api_version=0.40.0+1.17
+minecraft_version=1.18
+yarn_mappings=1
```

Comment:
Would like to switch to Quilt Mappings tbh

Reply:
Would you like me to do it? I don't know anything about quilt, but the migration is probably documented somewhere. Or will you do it yourself?

Reply:
I will do it myself, QM is a bit behind on some names that are used in this project.